src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsrestrictedpatientsapi/config/HmppsRestrictedPatientsApiExceptionHandler.kt

```diff
@@ -15,7 +15,7 @@ import org.springframework.web.client.RestClientException
 import org.springframework.web.client.RestClientResponseException
 import org.springframework.web.method.annotation.MethodArgumentTypeMismatchException
 import org.springframework.web.reactive.function.client.WebClientResponseException
-import uk.gov.justice.digital.hmpps.hmppsrestrictedpatientsapi.model.exceptions.NoResultsReturnedException
```

Comment:
Dropped this custom exception - it behaves exactly like t he EntityNotFoundException so using that instead.